storops/vnx/enums.py

```diff
@@ -261,6 +261,7 @@ class VNXRaidType(VNXEnum):
     RAID6 = 'r6'
     RAID10 = 'r1_0'
     HOTSPARE = 'hot_spare'
+    UNBOUND = 'Unbound'
```

Comment:
Can you please add a assert for this enum?

Reply:
Added/Updated corresponding test cases